### PR TITLE
[ser2sock] migrate to common library

### PR DESCRIPTION
**Pull request by onedr0p**
**Description of the change**

Migrate ser2sock to the common library

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [x] (optional) Variables are documented in the README.md

### CODE REVIEW

**Comment by onedr0p** [comment]
This chart is one we exclude, from testing because it requires a mounted device to pass tests. @billimek I don't use this, just going thru updating some charts to common. Let me know what you think!

**Comment by billimek** [comment]
It's really too bad there isn't a way to run this such that it fails if there isn't a physical device present.  Long ago I looked for an alternative container that does a similar thing and came up empty.  This does serve a useful purpose too.

File: charts/stable/ser2sock/values.yaml [hunk 1]
```diff
@@ -1,73 +1,55 @@
...
-nodeSelector: {}
+strategy:
+  type: Recreate
 
-# -- Toleration labels for pod assignment of the GUI
-tolerations: []
+env:
+  # TZ:
+  # PUID:
+  # PGID:
```
Comment (billimek) [inline charts/stable/ser2sock/values.yaml:19]: I'm OK with leaving these unset in the new version, but we had these specifically set before in the values to default to  `1001`.

Do we want to keep that default behavior, or leave it undefined, as a best practice?

Comment (onedr0p) [reply charts/stable/ser2sock/values.yaml:19]: Usually we've been keeping them commented out as it really depends on the peoples setup.

**Review by billimek — DISMISSED** [review]
LGTM!

**Comment by onedr0p** [comment]
@bjw feel free to merge unless you want to test this in the CI

**Comment by bjw-s** [comment]
Yes, I'll use this PR to fix that CI thingy, will merge afterwards 👍

**Pull request merged**